haas/rest.py

```diff
@@ -186,18 +195,35 @@ def _do_validation(schema, kwargs):
     `ValidationError`.
     """
 
-    if flask.request.data == '':
-        # No request body
-        final_kwargs = {}
+    final_kwargs = {}
+
+    if flask.request.method == "GET":
+        # GET requests can use URL and query parameter arguments
```

Comment:
s/URL/path/

the URL is the whole business:

    https://foo.bar.com:5353/some/path/elements?baz=quux&name=bob

The path is just:

    /some/path/elements

The same goes for the comment below.